### DISCUSSION TITLE
StreetComplete is...

### DISCUSSION THREAD

**Discussion by lgommans** [streetcomplete/StreetComplete#5259] [Feedback]
- Good-looking with the clean UI and helpful pictures or illustrations at every quest
- Something I can recommend to anyone interested in OSM editing without having to think if it will suit their needs, privacy requirements, technical knowledge...
- A nice way to introduce people to how OpenStreetMap editing works, like a friend or a family member who probably will never create an OSM account but are still superficially interested in what this hobby is. Saying "it's like wikipedia for maps" is a bit abstract and by opening StreetComplete there is an instant example of what it really means and how that works. Not sure it was meant as educational tool, but it works ^^
- Something I actively reach for, similar to how I had a habit of opening reddit (back when my favorite app still worked). The difference, of course, is that StreetComplete requires you to go outside or recently have been in a place that you haven't yet mapped to death, but requiring me to go outside is good for variety in my day :D
- Did I mention that being available on a store other than The One Which Shall Not Be Named is a big benefit as well? No need to deal with "is Aurora rate-limited today?" "did google kill the anonymous accounts again?" "shall the great lord of the dark allow my access to apks today?" No, you just grab it from fdroid
- Inspirational for anyone who wants to make technical operations, like OSM object manipulation, into something anyone can start doing instantly
- Useful, helpful, valuable to so many people, as in, don't only think of mappers who literally find the app useful, also think of everyone who then uses the resultant data enhancements in a variety of geodata applications or research that can use OSM data, and they are unlikely to know the role StreetComplete played in enabling that level of consistency and detail for various objects

Thank you for an app so good that I really find it inspirational :smiley:

**Comment by lgommans** [comment]
I didn't ping names because I didn't know who exactly is involved enough with StreetComplete to address this to. The idea was that anyone currently active might peek into the discussions and see it. It looks like that mostly worked, but I don't know if the project founder saw it and they're of course one of the persons whom this is directed at!  
@westnordost hi! Please know that you, and the others that made StreetComplete into the helpful tool it is, are appreciated :D

**Reply by westnordost** [reply]
I have notifications off for new tickets, so I didn't notice it. Thank you for the ping, and for the praise 😃.

I really appreciate your text and am glad that you like the app! Makes me think that I should also do this more frequently with projects I love. ❤
